sct.py

```diff
@@ -362,5 +369,168 @@ def cloud_usage_report(emails):
     click.secho(message="Done." % email_list, fg="yellow")
 
 
+@cli.command('collect-logs', help='Collect logs from cluster by test-id')
+@click.option('--test-id', help='Find cluster by test-id')
+@click.option('--logdir', help='Path to directory with sct results')
+@click.option('--backend', help='Clound where search nodes', default='aws')
+# @click.option('--user', help='Authenticate with user to remote nodes')
+# @click.option('--key_file', help='Key file for authorization')
+# def collect_logs(test_id=None, backend='aws', user='centos', key_file='', log_dir=None):
+def collect_logs(test_id=None, logdir=None, backend='aws'):
+    from sdcm.logcollector import Collector
+    if not os.environ.get('SCT_CLUSTER_BACKEND', None):
+        os.environ['SCT_CLUSTER_BACKEND'] = backend
+
+    config = SCTConfiguration()
+
+    collector = Collector(test_id=test_id, params=config, test_dir=logdir)
+
+    collected_logs = collector.run()
+
+    table = PrettyTable(['Cluster set', 'Link'])
+    table.align = 'l'
+    for cluster_type, s3_link in collected_logs.items():
+        table.add_row([cluster_type, s3_link])
+    click.echo(table.get_string(title="Collected logs by test-id: {} Storing dir: {}".format(collector.test_id,
+                                                                                             collector.storing_dir,)))
+
+
+@cli.command('destroy-resources', help='')
+@click.option('--logdir', type=str)
+@click.option('--backend', type=str)
+# pylint: disable=too-many-statements
```

Comment:
please put those comment inside the function, here they affect the rest of the file

Reply:
done